src/Avalonia.Themes.Default/Slider.xaml

```diff
@@ -72,8 +72,7 @@
   <Style Selector="Slider /template/ Track#PART_Track">
     <Setter Property="Minimum" Value="{TemplateBinding Minimum}"/>
     <Setter Property="Maximum" Value="{TemplateBinding Maximum}"/>
-    <Setter Property="Value" Value="{TemplateBinding Path=Value, Mode=TwoWay}"/>
-    <Setter Property="Orientation" Value="{TemplateBinding Orientation}"/>
+    <Setter Property="Value" Value="{TemplateBinding Value, Mode=TwoWay}"/>
```

Comment:
This setter was both unnecessary and caused problems in that it was binding to `Slider.Orientation` instead of `Track.Orientation` which are actually different `AvaloniaProperty`s (#1696). This was resulting in the `Slider` getting messed up.